models/Asset/Video.php

```diff
@@ -139,7 +139,7 @@ public function getThumbnail($thumbnailName, $onlyFormats = [])
      */
     private function enrichThumbnailPath($path)
     {
-        $fullPath = rtrim($this->getRealPath(), '/') . $path;
+        $fullPath = $this->getRealPath() . ltrim($path, '/');
```

Comment:
```suggestion
        $fullPath = rtrim($this->getRealPath(), '/') . '/' . ltrim($path, '/');
```
Maybe we should keep both here, to ensure the right format?